python/mxnet/contrib/quantization.py

```diff
@@ -552,9 +552,9 @@ def quantize_model_mkldnn(sym, arg_params, aux_params, data_names=('data',),
         raise ValueError('currently only supports single ctx, while received %s' % str(ctx))
     if ctx.device_type != 'cpu':
         raise ValueError(
-            'quantize_model_mkldnn only support Intel cpu platform with MKL-DNN Backend')
+            'quantize_model_dnnl only support Intel cpu platform with DNNL Backend')
```

Comment:
```suggestion
            'quantize_model_dnnl only support Intel cpu platform with oneDNN Backend')
```

Reply:
I would suggest to use `quantize_model_onednn` instead of `quantie_model_dnnl`. There's no need to introduce naming with `dnnl` unless it was already there.